### ISSUE TITLE
Design: Page for /employers

### ISSUE THREAD

**Issue by dmarchante** [OperationCode/front-end#80]
# Feature
Similar to a product demo, an employers demo will be a landing for companies, organizations and government agencies that want to know more about hiring our military veterans, spouses and military family members. 

## Why is this feature being added?
This feature enables employers to quickly and seamlessly learn more about our organization and determine whether hiring one of our members makes sense. The value add is that CEOs, C-level and hiring managers don't have to wait to get in touch, they can have all the data at their fingertips at an instance and make an informed decision internally before reaching out. We want to limit the warming period as much as possible. We want employers to know the value add before reaching out. 

## What should your feature do?
- The landing (or popup) will ask for name, title, company, email and phone number to request more information. This data can populate back into our Airtable where the CEO and/or Chief of Staff can make contact and answer their questions and next steps.
- The automatic email response will send the recipient our short deck, and a link to a visualization graph of our membership, including (1) total number in our community (via Slack), (2) geographic location (via zip code), (3) military branch, (4) number of employed, (5) number underemployed/unemployed and actively looking for a software developer role, and (6) programming languages. 
- Internally, we should be able to update this and automate as much as possible. Internally, we should have a special code to access the visualization graph (or make it open) to brief folks when we're giving a talk or in a meeting.

**Comment by dmarchante** [comment]
This issue blocked by the need for parity.

**Comment by DEBSUBHRO** [comment]
Can I suggest a design/mock?

**Comment by kylemh** [comment]
Yes please. Think of it as a similar interface to the code schools page.

**Comment by LindaCBeall** [comment]
@dmarchante can this be unblocked?

**Comment by kylemh** [comment]
It's unfortunately still blocked by both a need for design as well as a need for data - it may or may not require back-end work.

**Comment by kylemh** [comment]
This one is considered a lower priority. We want to recreate a new ticket around this concept once it's the priority.
